lib/instrumentation/aws-sdk/v3/redshift.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const { setRedshiftParameters } = require('../util')
+const { OperationSpec } = require('../../../shim/specs')
+const InstrumentationDescriptor = require('../../../instrumentation-descriptor')
+
+function getRedshiftSpec(shim, original, name, args) {
+  const [{ input }] = args
+  return new OperationSpec({
+    name: this.commandName,
+    parameters: setRedshiftParameters(this.endpoint, input),
+    callback: shim.LAST,
+    opaque: true,
+    promise: true
+  })
+}
+
+async function getEndpoint(config) {
+  if (typeof config.endpoint === 'function') {
+    return await config.endpoint()
+  }
+
+  const region = await config.region()
+  return new URL(`https://redshift-data.${region}.amazonaws.com`)
```

Comment:
why is this hardcoded?

Reply:
ok so i see that we do this for dynamodb as well. Looking at the library code for both looksl ike the URLs are: `https://redshift-data.{Region}.{PartitionResult#dualStackDnsSuffix}` or `https://redshift-data-fips.{Region}.{PartitionResult#dualStackDnsSuffix}`.  Same goes for dynamodb. this is fine as it'll rely on the result of `config.endpoint()`